test/Datadog.Trace.Tests/TracerTests.cs

```diff
@@ -128,7 +129,7 @@ public void StartActive_NoServiceName_DefaultServiceName()
         {
             var scope = _tracer.StartActive("Operation");
 
-            Assert.Equal("testhost", scope.Span.ServiceName);
+            Assert.Contains(scope.Span.ServiceName, TestRunners.ValidNames);
```

Comment:
My only concern is that in order to check that the `ServiceName` is one of a set of valid names you've got to reverse the order of parameters to this assert, which could be confusing for someone used to the first parameter being the "expected" value (which is what happened when I first read this code).

Unfortunately, I couldn't find anything like `Assert.IsIn` in the xunit code.  If there's a lot of tests like this, I'd consider writing a helper method that would maintain the proper order of : `expected` and `actual`.